dask_jobqueue/sge.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import os
+import sys
+
+from .core import JobQueueCluster
+
+logger = logging.getLogger(__name__)
+
+dirname = os.path.dirname(sys.executable)
+
+
+class SGECluster(JobQueueCluster):
+    """ Launch Dask on a SGE cluster
+
+    Parameters
+    ----------
+    name : str
+        Name of worker jobs. Passed to `$SGE -N` option.
+    queue : str
+        Destination queue for each worker job. Passed to `#$ -q` option.
+    project : str
+        Accounting string associated with each worker job. Passed to
+        `#$ -A` option.
+    threads_per_worker : int
+        Number of threads per process.
+    processes : int
+        Number of processes per node.
+    memory : str
+        Bytes of memory that the worker can use. This should be a string
+        like "7GB" that can be interpretted both by SGE and Dask.
+    resource_spec : str
+        Request resources and specify job placement. Passed to `#$ -l`
+        option.
+    walltime : str
+        Walltime for each worker job.
+    interface : str
+        Network interface like 'eth0' or 'ib0'.
+    death_timeout : float
+        Seconds to wait for a scheduler before closing workers
+    extra : str
+        Additional arguments to pass to `dask-worker`
+    kwargs : dict
+        Additional keyword arguments to pass to `LocalCluster`
+
+    Examples
+    --------
+    >>> from dask_jobqueue import SGECluster
+    >>> cluster = SGECluster(project='...')
+    >>> cluster.start_workers(10)  # this may take a few seconds to launch
+
+    >>> from dask.distributed import Client
+    >>> client = Client(cluster)
+
+    This also works with adaptive clusters.  This automatically launches and
+    kill workers based on load.
+
+    >>> cluster.adapt()
+    """
+
+    #Override class variables
+    submit_command = 'qsub -terse'
+    cancel_command = 'qdel'
+
+    def __init__(self,
+                 queue=None,
+                 project=None,
+                 resource_spec=None,
+                 walltime='0:30:00',
+                 interface=None,
```

Comment:
I believe interface=None should not be here. It is a keyword from the parent class, and you don't seem to use it.

Reply:
Good catch, done.